### ISSUE TITLE
Rides changing colors

### ISSUE THREAD

**Issue by AutumnStorme** [OpenRCT2/OpenRCT2#17531]
### Operating System

Ubuntu 20.04

### OpenRCT2 build

OpenRCT2, v0.4.1[DEBUG]

### Describe the issue

I have not installed a plugin that should be causing this issue to my knowledge.

Every single time I start a game I've saved, or build a new park, all of my rides (coasters, flat rides, everything) change colors from what they're supposed to be.  I can manually change it back to the color it's supposed to be, but saving it does nothing to keep it from happening again.

I'm not sure what I need to provide here other than this - I've never reported a bug before.  I included two screenshots from my current build - the top is right when I open, the bottom is what color the rides are supposed to be.

### Area(s) with issue?

Building the game

### Steps to reproduce

1. 
2. 
3. 

### Attachments

![Amity Airfield 2022-07-09 16-28-48](https://user-images.githubusercontent.com/109000190/178121705-1969f0b1-c513-4a85-8b41-5a2151252bd7.png)
![Amity Airfield 2022-07-09 16-30-43](https://user-images.githubusercontent.com/109000190/178121706-25971ab2-ca5b-45ab-9bb6-6c699cc24771.png)

**Comment by janisozaur** [comment]
If you load a single save multiple times, do you get different colour each time? Try doing a test with cycling through several parks (load park 1, load park 2, load park 1, load park 3, load park 1) - and report ride colours.

We will also need better version information - we know Linux builds don't provide that easily, but you can take version from package you installed from.

**Comment by AutumnStorme** [comment]
> If you load a single save multiple times, do you get different colour each time? Try doing a test with cycling through several parks (load park 1, load park 2, load park 1, load park 3, load park 1) - and report ride colours.
> 
> We will also need better version information - we know Linux builds don't provide that easily, but you can take version from package you installed from.

Yes, it gives me different colors every single time.  I can turn it back to the colors I want, save it, and when I open it again it'll be a different color than the previous.  Or I can just exit to the menu and open again and it'll be a different color.

If someone can explain to me like I'm 5 how to get the build information, I'll be happy to provide it.  I swapped to Linux several years back, but I still don't know a whole lot about it.

**Comment by janisozaur** [comment]
Where did you get openrct2 from?

**Comment by AutumnStorme** [comment]
> Where did you get openrct2 from?

From the source: https://openrct2.org/downloads

**Comment by janisozaur** [comment]
"from the source" would imply you have cloned the repo and built it.

Can you provide the commit for which you've downloaded build if `develop` or tell if you've used release?

**Comment by AutumnStorme** [comment]
> "from the source" would imply you have cloned the repo and built it.
> 
> Can you provide the commit for which you've downloaded build if `develop` or tell if you've used release?

You asked me where I got the game from.  I gave you the exact site I downloaded it from.  I downloaded the newest release, uninstalled the prior version (which was working fine), then installed the newest version that I downloaded FROM OpenRCT2 site.  My game does not do updates, so I had to uninstall and then reinstall the newest updated version.

**Comment by janisozaur** [comment]
Can you try on latest `develop` build then?

**Comment by AutumnStorme** [comment]
> Can you try on latest `develop` build then?

That's what I'm using.

**Comment by janisozaur** [comment]
> > Can you try on latest `develop` build then?
> 
> That's what I'm using.

And earlier:

> I downloaded the newest release.

Which one is it then? And if it is develop, you still haven't provided commit.

**Comment by AutumnStorme** [comment]
I gave you what version I'm using.  It's this exact one.  Version 0.4.1 - the link: https://github.com/Limetric/OpenRCT2-binaries/releases/download/v0.4.1-95bcca3/OpenRCT2-0.4.1-develop-95bcca3-linux-x86_64.tar.gz

That is the newest release, the newest version, the latest update, whatever you want to call it.  This is **exactly** what I downloaded to reinstall.  The original install was working fine for me, no color changes, no weirdness, nothing.  I completely removed it, and reinstalled **this version** which I have told you now and linked you to the exact download as well.  From GitHub.  Again, this is the latest and greatest update/release/version that everyone is running right now, the one with the alpine coaster.

**Comment by cheweytoo** [comment]
@AutumnStorme: Calm down a bit.

To clarify: releases and development builds are different things.

Releases are those: https://openrct2.org/downloads/releases - and yours isn't from there.

Development builds are those: https://openrct2.org/downloads/develop - Those are NOT official releases, they are built from the current development code base, may contain new/unexpected issues, and "newest dev build" changes up to several times a day. That's why you've been asked to provide the exact version of the build you're using.

"which commit?" then refers to the commit ID, 95bcca3 in your case.
